nixos/modules/programs/steam.nix

```diff
@@ -53,8 +53,8 @@ in {
       driSupport32Bit = true;
     };
 
-    # optionally enable 32bit pulseaudio support if pulseaudio is enabled
-    hardware.pulseaudio.support32Bit = config.hardware.pulseaudio.enable;
+    # optionally enable 32bit pulseaudio support if pulseaudio is available
+    hardware.pulseaudio.support32Bit = config.hardware.pulseaudio.available;
```

Comment:
Right, so this change is a no-op. I'll revert it.

> set `services.pipewire.alsa.support32Bit` instead.

That will be up to the user. I don't think generalizing this would be particularly helpful, so I don't think this PR can help with that.